### ISSUE TITLE
iOS 13.2 Hanging

### ISSUE THREAD

**Issue by chrisshay** [openid/AppAuth-iOS#470]
I am using this library in my app and when I run my app on an iOS 13.2 device by login just hangs. 

When I run my app on an iOS 13.1 device it is fine, logs in without an issue. 

However on iOS 13.2 I am directed to my sign in page correctly, however when I return from my login page my app just hangs - it is as if it never receives the callback response. 

Has anyone else encountered this issue on iOS13.2?

This also happens on the iOS 13.3 Beta

I will post more detail once I have investigated, I just wanted to reach out.

**Comment by julienbodet** [comment]
I didn't experience this issue on my side. I tested with the example provided and everything worked as expected.

@chrisshay what are the results of your investigation?

**Comment by chrisshay** [comment]
@julienbodet After my investigation I found that you are correct, the issue with my app was not with the AppAuth Code, it gave the impression that it was however I could see that all of the authentication calls were firing as expected. 

The issue was with some of the UI threads in my application, I am not 100% sure what has changed since iOS 13.1 but these threads were getting lost and my app was never recovering. This is a separate issue that has been resolved.

I will close this issue, thanks for responding.
